README.md

```diff
@@ -88,6 +88,25 @@ includes builds of this Go fork.
 * In Azure Linux 2.0, the package `msft-golang` installs this fork.
 * In Azure Linux 3.0, the `golang` package installs this fork.
 
+### Ubuntu
+
+To install the Microsoft Build of Go using an Ubuntu package, open a terminal and run the following commands:
```

Comment:
```suggestion
To install the Microsoft build of Go using an Ubuntu package, open a terminal and run the following commands:
```

(For some reason we have the "Microsoft Build of OpenJDK" and "Microsoft build of Go". 🤷)